website/docs/reference/operators/comparison-operators.md

```diff
@@ -0,0 +1,385 @@
+    name: { first: 'Earl', last: 'Thomas' },
+    employeeID: 1234,
+    age: 23,
+    role: 'salesperson',
+    catalog: [ 'printer', 'cardboard', 'crayons', 'books' ]
+  },
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0d"),
+    name: { first: 'Sam', last: 'Johnson' },
+    employeeID: 2234,
+    age: 35,
+    role: 'salesperson',
+    catalog: [ 'cabinet', 'fridge', 'blender', 'utensils' ]
+  }
+]
+```
+
+## $gte
+
+*Syntax*: `{ field: { $gte: value } }`
+
+The greater than or equal operator `$gte`.
+The following operation selects documents based on the specified query, where the field `age` is greater than or equal to `21`.
+
+```js
+db.employees.find({
+   age: {
+      $gte: 21
+   }
+})
+```
+
+The output:
+
+```js
+[
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0c"),
+    name: { first: 'Earl', last: 'Thomas' },
+    employeeID: 1234,
+    age: 23,
+    role: 'salesperson',
+    catalog: [ 'printer', 'cardboard', 'crayons', 'books' ]
+  },
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0d"),
+    name: { first: 'Sam', last: 'Johnson' },
+    employeeID: 2234,
+    age: 35,
+    role: 'salesperson',
+    catalog: [ 'cabinet', 'fridge', 'blender', 'utensils' ]
+  },
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0e"),
+    name: { first: 'Clarke', last: 'Dane' },
+    employeeID: 3234,
+    age: 21,
+    role: 'salesperson',
+    catalog: [ 'printer', 'pencils', 'crayons', 'toys' ]
+  }
+]
+```
+
+## $lt
+
+*Syntax*: `{ field: { $lt: value } }`
+
+The less than operator `$lt`.
+The following operation queries for documents where the field `age` is less than `25`.
+
+```js
+db.employees.find({
+   age: {
+      $lt: 25
+   }
+})
+```
+
+The output:
+
+```js
+[
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0c"),
+    name: { first: 'Earl', last: 'Thomas' },
+    employeeID: 1234,
+    age: 23,
+    role: 'salesperson',
+    catalog: [ 'printer', 'cardboard', 'crayons', 'books' ]
+  },
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0e"),
+    name: { first: 'Clarke', last: 'Dane' },
+    employeeID: 3234,
+    age: 21,
+    role: 'salesperson',
+    catalog: [ 'printer', 'pencils', 'crayons', 'toys' ]
+  }
+]
+```
+
+## $lte
+
+*Syntax*: `{ field: { $lte: value } }`
+
+The less than or equal operator `$lte`.
+The following operation queries for documents where the field `age` is less than or equal to `21`.
+
+```js
+db.employees.find({
+   age: {
+      $lte: 21
+   }
+})
+```
+
+The output:
+
+```js
+[
+  {
+    _id: ObjectId("639a3cce071b6bed396a8f0e"),
+    name: { first: 'Clarke', last: 'Dane' },
+    employeeID: 3234,
+    age: 21,
+    role: 'salesperson',
+    catalog: [ 'printer', 'pencils', 'crayons', 'toys' ]
+  }
+]
+```
+
+## $in
+
+*Syntax*: `{ field: { $in: [<value1>, <value2>, ... <valueN> ] } }`
+
+The `$in` operator selects documents that contain any of the values in a specified array.
+The following operation queries the `employees` collection for documents where the `catalog` array contains `printer` and `crayons`.
```

Comment:
Should it be `printer` or `crayons`? If I recall, `$in` needs to match at least one element of the array. I think the example should show where only one of `printer` or `crayons` present. Maybe we can remove `printer` from one of the example and demonstrate the match. :hugs: If ours doesn't behave like that, maybe there is a bug.

Reply:
PTAL